assembly/pallets/balances/assembly/balances-module.ts

```diff
@@ -0,0 +1,99 @@
+import { AccountData, AccountId, ISignedTransaction, TransactionValidity, ResponseCodes } from 'subsembly-core';
+import { Storage, Log} from "subsembly-core";
+import { ByteArray, BytesReader, UInt128, UInt64 } from "as-scale-codec";
+import { u128 } from "as-bignum";
+import { System } from "../../../frame";
+import { AccountIdType } from "../../../runtime/runtime";
+
+/**
+ * The Balances Module.
+ * Used for account balance manipulation such as:
+ *  - Getting and setting free/reserved balances
+ */
+export class Balances {
```

Comment:
I think that we should think about providing the possibility for users to define their own types for balance (not u128 f.e)